### PR TITLE
Fake console for simple Ethernet switch

### LINKED ISSUE #454: Fake console for simple Ethernet switch
The idea is to expose the MAC address table via a command but we can potentially add more stuff too. This could be applicable to other type of device like Openvswitch etc. and may also be extremely useful for debugging purposes.

This can be implemented using the async Telnet server  https://github.com/GNS3/gns3-server/blob/1.5/gns3server/utils/asyncio/telnet_server.py

### PR DESCRIPTION
**Pull request by noplay**
Fix #454

### CODE REVIEW

File: gns3server/compute/dynamips/nodes/ethernet_switch.py [hunk 1]
```diff
@@ -115,6 +164,12 @@ def create(self):
 
         yield from self._hypervisor.send('ethsw create "{}"'.format(self._name))
         log.info('Ethernet switch "{name}" [{id}] has been created'.format(name=self._name, id=self._id))
+
+        self._telnet_shell = EthernetSwitchConsole(self)
+        self._telnet_shell.prompt = self._name + '> '
```
Comment (grossmj) [inline gns3server/compute/dynamips/nodes/ethernet_switch.py]: This should be set via a method.

**Comment by grossmj** [comment]
There is a problem to drag & drop nodes in this PR:

I get these messages: ```Error while creating node: Appliance a6b3e96a-5f79-4329-b921-76f9d03523df doesn't exist```

or no error and the node doesn't show up on the drawing area.

I do not have these issues on the 2.1 branch.

File: gns3server/compute/dynamips/nodes/ethernet_switch.py (hunk 1 above)
Comment (grossmj) [inline gns3server/compute/dynamips/nodes/ethernet_switch.py]: Actually this is a property... I guess this is fine.

**Comment by noplay** [comment]
What kind of node trigger your error?

**Comment by grossmj** [comment]
Any node, Ethernet switch etc.

**Comment by noplay** [comment]
I don't replicate it :(

**Comment by grossmj** [comment]
I suspect this is due to aiohttp dependency again...

I tested in a virtual env and I have the same problems.

```
aiohttp==2.0.7
aiohttp-cors==0.5.3
async-timeout==1.2.1
chardet==3.0.2
Jinja2==2.9.6
jsonschema==2.6.0
MarkupSafe==1.0
multidict==2.1.5
pkg-resources==0.0.0
psutil==5.2.2
PyQt5==5.8.2
raven==6.0.0
sip==4.19.2
typing==3.6.1
yarl==0.10.2
zipstream==1.1.4
```

**Comment by noplay** [comment]
Could you update both branch? I have merge last 2.1 code to it

**Comment by grossmj** [comment]
I still have the problem. Create a new project and drag&drop a simple Ethernet switch and I get "Error while creating node: Appliance dff1efa4-c9f7-460e-a576-6f4f15049027 doesn't exist".

I do not have the problem when a load an existing project.

This problem is present in 2.1 branch too.

**Comment by grossmj** [comment]
Tried again in a clean environment, same issue on 2.1 branches.

```
python3 -m venv gns3-venv
source gns3-venv/bin/activate
pip3 install pyqt5
cd gns3-gui
python3 setup.py install
cd ..
cd gns3-server
python3 setup.py install
```

**Comment by noplay** [comment]
If you reset the settings you stil have the problem?

**Comment by grossmj** [comment]
Same problem with reset settings.

Server side:

```2017-06-09 10:21:46 ERROR project.py:331 Appliance 9d24a99a-def8-451a-ac0b-78e7b1f484a4 doesn't exist```

GUI side:

```2017-06-09 10:21:46 ERROR appliance_manager:98 Error while creating node: Appliance 9d24a99a-def8-451a-ac0b-78e7b1f484a4 doesn't exist```

**Comment by noplay** [comment]
I will boot a second machine to test

**Comment by noplay** [comment]
I don't understand I don't have the issue on my ubuntu box :(  You use the branch ethernet_switch_console for the GUI with the last commits?

**Comment by grossmj** [comment]
I've noticed the bug can be random... excepting here it happens almost all the time.

**Comment by grossmj** [comment]
Please merge the appliance not found fix and I will test that PR.

**Comment by noplay** [comment]
Merged the 2.1 branch

**Pull request merged**